src/index.ts

```diff
@@ -10,8 +10,8 @@ async function main(): Promise<void> {
 
 main().then()
     .catch((error) => {
-        core.setOutput('imagebuilderRunStatus', "failed");
+        console.log("$(imagebuilder-run-status) = ", "failed");
+        core.setOutput('imagebuilder-run-status', "failed");
```

Comment:
Won't the log be printed twice?

Reply:
nhi..core.setOutput('imagebuilder-run-status', "failed"); wont be printed as part of logs